missioncontrol/etl/measure.py

```diff
@@ -113,14 +102,46 @@ def update_measures(application_name, platform_name, channel_name,
         'os_name': platform.telemetry_name,
         'channel_name': channel_name,
         'min_timestamp': min_timestamp.strftime("%Y-%m-%d %H:%M:%S"),
-        'submission_date': submission_date.strftime("%Y%m%d")
+        'submission_date': submission_date.strftime("%Y-%m-%d")
     }
-    logger.info('Querying: %s', query_template % params)
+
+    bq_query = f'''
```

Comment:
Nit: I feel like `query_sql` might be a bit more of a descriptive variable name here.